code/Test_definitions/customer-insights-retrieveScoring.feature

```diff
@@ -206,27 +206,101 @@ Feature: CAMARA Customer Insights API, v0.2.0-rc.1 - Operation retrieveScoring
+  And the request body property "$.phoneNumber" is set to a valid phone number existing in the environment
+  And the request body property "$.idDocument" is set to a valid value but not supported by Telco Operator
+  And the request body property "$.scoringType" is set to "veritasIndex"
+  When the request "retrieveScoring" is sent
+  Then the response status code is 422
+  And the response property "$.status" is 422
+  And the response property "$.code" is "CUSTOMER_INSIGHTS.ID_DOCUMENT_NOT_SUPPORTED"
+  And the response property "$.message" contains a user friendly text
+
+@retrieve_scoring_422.04a_id_document_required_three_legged
+Scenario: Id Document required with 3-legged token
+  Given the header "Authorization" is set to a valid access token identifying a phone number
+  And the request body property "$.idDocument" is not included
+  And the request body property "$.phoneNumber" is not included
+  And the Telco Operator business rules require idDocument for the identified phone number
+  When the request "retrieveScoring" is sent
+  Then the response status code is 422
+  And the response property "$.status" is 422
+  And the response property "$.code" is "CUSTOMER_INSIGHTS.ID_DOCUMENT_REQUIRED"
+  And the response property "$.message" contains a user friendly text
+
+@retrieve_scoring_422.04b_id_document_required_two_legged
+Scenario: Id Document required with 2-legged token
+  Given the header "Authorization" is set to a valid access token which does not identify a single phone number
+  And the request body property "$.phoneNumber" is set to a valid phone number existing in the environment
+  And the request body property "$.idDocument" is not included
+  And the Telco Operator business rules require idDocument for the provided phone number
+  When the request "retrieveScoring" is sent
+  Then the response status code is 422
+  And the response property "$.status" is 422
+  And the response property "$.code" is "CUSTOMER_INSIGHTS.ID_DOCUMENT_REQUIRED"
+  And the response property "$.message" contains a user friendly text
+
+@retrieve_scoring_422.04c_id_document_required_three_legged_with_scoring_type
+Scenario: Id Document required with 3-legged token and scoringType
+  Given the header "Authorization" is set to a valid access token identifying a phone number
+  And the request body property "$.idDocument" is not included
+  And the request body property "$.scoringType" is set to "gaugeMetric"
+  And the request body property "$.phoneNumber" is not included
+  And the Telco Operator business rules require idDocument for the identified phone number
+  When the request "retrieveScoring" is sent
+  Then the response status code is 422
+  And the response property "$.status" is 422
+  And the response property "$.code" is "CUSTOMER_INSIGHTS.ID_DOCUMENT_REQUIRED"
+  And the response property "$.message" contains a user friendly text
+
+@retrieve_scoring_422.04d_id_document_required_two_legged_with_scoring_type
```

Comment:
Same comments as for test: 
@retrieve_scoring_422.03c_id_document_not_supported_three_legged_with_scoring_type